roles/ceph-client/tasks/create_users_keys.yml

```diff
@@ -12,6 +12,16 @@
   when:
     - keys_tmp is defined
 
+# dummy container run is currently not supported for ppc64le
+# this task creates a group that contains only non-ppc64le clients
+- name: create filtered clients group
+  add_host:
+    name: "{{ item }}"
+    groups: _filtered_clients
+  with_items: "{{ groups[client_group_name] }}"
+  when:
+    - hostvars[item]['ansible_architecture'] != 'ppc64le'
```

Comment:
I wonder if it's worth expressing this as:

`- hostvars[item]['ansible_architecture'] == 'x86_64'`

that way if someone tries to do the client setup on aarch64 they don't run into the same issue.

Reply:
+1